clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/provider/agent/AmazonApplicationLoadBalancerCachingAgent.groovy

```diff
@@ -298,12 +473,75 @@ class AmazonApplicationLoadBalancerCachingAgent extends AbstractAmazonLoadBalanc
         }
       }
     }
+
+    List<InstanceTargetGroupState> itgStates = []
+    for (TargetGroup tg : allTargetGroups) {
+      // Collect health information for the target group and instance ids
+      List<String> instanceIds = new ArrayList<String>()
+      List<TargetHealthDescription> thds = targetGroupArnToHealths.get(tg.targetGroupArn)
+      for (TargetHealthDescription thd : thds) {
+        itgStates << new InstanceTargetGroupState(thd.target.id, ArnUtils.extractTargetGroupName(tg.targetGroupArn).get(), thd.targetHealth.state, thd.targetHealth.reason, thd.targetHealth.description)
```

Comment:
crazy long line, worth getting in the habit of splitting 

```
itgStates << new InstanceTargetGroupState(
  thd.target.id,
  ArnUtils...(),
  ...
)
```

Matter of personal preference but I also don't like shortening the variable names ... `thds` but 🤷‍♂️